src/apps/concepts/pages/CreateOrEditConceptPage.tsx

```diff
@@ -236,7 +243,8 @@ const mapStateToProps = (state: AppState) => ({
 const mapActionsToProps = {
   retrieveDictionary: makeRetrieveDictionaryAction(true),
   retrieveConcept: retrieveConceptAction,
-  upsertConcept: upsertConceptAndMappingsAction
+  upsertConcept: upsertConceptAndMappingsAction,
+  resetConceptForm: resetConceptFormAction
```

Comment:
Move the `resetConceptForm` line to the top of this block so it's in alphabetical order again.